src/components/Navigation/Navigation.scss

```diff
@@ -229,92 +217,41 @@ $disabled-fade: 0.6;
     height: nav(desktop-height);
   }
 
-  .Navigation-newDesignLanguage & {
-    @include recolor-icon(
-      var(--p-icon-subdued, color('ink', 'lightest')),
-      var(--p-override-transparent, color('white')),
-      filter('icon')
-    );
-  }
-
-  &:hover {
+  &:hover,
+  &:focus,
+  &:active {
     @include recolor-icon(
-      var(--p-action-primary, color('indigo')),
+      var(--p-icon-hovered, color('indigo')),
       var(--p-override-transparent, color('white')),
       filter('indigo')
     );
-    background-image: var(
-      --p-override-none,
+    background: var(
+      --p-background-hovered,
       linear-gradient(color(state, hover), color(state, hover))
     );
-    background-color: var(--p-surface-hovered);
 
     .Navigation-newDesignLanguage & {
-      @include recolor-icon(
-        var(--p-action-primary, color('indigo')),
-        var(--p-override-transparent, color('white')),
-        filter('action')
-      );
+      @media (-ms-high-contrast: active) {
+        @include recolor-icon(
+          $fill-color: color('white'),
+          $filter-color: filter('white')
+        );
+      }
```

Comment:
Small tweak to make icons easier to spot in Edge/Win10 on hover. Sadly Firefox turns them into black and they become really hard to spot on hover in that browser.